From a2e39b1243bc013827d76ccce5c631b04442458e Mon Sep 17 00:00:00 2001
From: Enno Boland <gottox@voidlinux.org>
Date: Wed, 19 Aug 2015 10:41:04 +0200
Subject: [PATCH 2/3] Add missing include

The limits.h header must be included for PATH_MAX.

Fixes build against musl libc.

Origin: Void Linux
Upstream-Status: Unknown
[ismael@sourcemage.org: Edited description]
Signed-off-by: Ismael Luceno <ismael@sourcemage.org>
---
 src/utils.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/utils.c b/src/utils.c
index 2497cf575045..d78a38fd5b8e 100644
--- a/src/utils.c
+++ b/src/utils.c
@@ -22,6 +22,7 @@
 #include <fcntl.h>
 #include <signal.h>
 #include <libintl.h>
+#include <limits.h>
 
 /* File name used to tag directories created by pmount */
 #define CREATED_DIR_STAMP ".created_by_pmount"
-- 
2.40.1

